### PR TITLE
Improve editor scrolling speed to be adapted based on zoom

### PR DESCRIPTION
**Pull request by ClementPasteau**
Addresses #3226 

- increase default speed
- add a speed factor based on zoom

https://user-images.githubusercontent.com/4895034/142996573-74c858cc-ef15-4a1a-b634-7adc0294b269.mov

### CODE REVIEW

File: newIDE/app/src/InstancesEditor/index.js [hunk 1]
```diff
@@ -167,9 +167,12 @@ export default class InstancesEditor extends Component<Props> {
       if (this.keyboardShortcuts.shouldZoom()) {
         this.zoomOnCursorBy(-event.deltaY / 5000);
       } else if (this.keyboardShortcuts.shouldScrollHorizontally()) {
-        this.viewPosition.scrollBy(-event.deltaY / 10, 0);
+        this.viewPosition.scrollBy(-event.deltaY / 5, 0);
```
Comment (4ian) [inline newIDE/app/src/InstancesEditor/index.js:170]: Is it normal not to use the zoomFactor here?

Comment (ClementPasteau) [reply newIDE/app/src/InstancesEditor/index.js:170]: I had not tested this behavior as it was never triggered with my tests.
But now I realise this happens when the user press "alt" and then scroll vertically -> trigger a horizontal scroll.
(I didn't know this was a common scrolling feature 🤔 )
I guess it should apply the same logic as well!

Comment (4ian) [reply newIDE/app/src/InstancesEditor/index.js:170]: I guess same logic yeah! 
This was adapted from software doing the same, especially useful for traditional mouses/no trackpad which don't have a horizontal scroll.

**Comment by Bouh** [comment]
So smoothhhh.
Is it also in point, mask, physics, and resources preview?

**Comment by ClementPasteau** [comment]
@Bouh This is not affecting those screens. 
I don't think this is necessary as the objects usually fit the screen well + the scrolling seems better handled there.
By testing it, I couldn't find any frustration, so leaving it as is :)

**Review by 4ian — APPROVED** [review]

**Pull request merged**